lib/cog/models/bundle_version.ex

```diff
@@ -23,4 +28,18 @@ defmodule Cog.Models.BundleVersion do
     model |> cast(params, @required_fields, [])
   end
 
+  def enabled?(bundle_version) do
+    if Ecto.assoc_loaded?(bundle_version.enabled_version_registration) do
+      if bundle_version.enabled_version_registration do
```

Comment:
Could this be `bundle_version.enabled_version_registration != nil` instead of the `if`?